ext/smtp/sink.go

```diff
@@ -150,7 +150,7 @@ func NewSink(commonSink common.Sink,
 	commonSink.Logger().Info(fmt.Sprintf("using smtp sink with storage: %s", storageConfig.Mode))
 
 	// prepare handlers
-	if storageConfig.Mode == "oss" {
```

Comment:
can we still use `oss` as lowercase? we can do string lowercase before passing this config in oss sink constructor

Reply:
makes sense